<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QThread_testClass</class>
 <widget class="QMainWindow" name="QThread_testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QThread_test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>传送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>读取</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>216</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="btnshow"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>btnSave</zorder>
   <zorder>btnSend</zorder>
   <zorder>btnshow</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QThread_test.qrc"/>
 </resources>
 <connections/>
</ui>
